addon/components/frost-link.js

```diff
@@ -174,30 +180,49 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
    * @returns {boolean} true if we need to open multiple links on click and false otherwise
    */
   _hasMultipleLinks () {
-    return get(this, 'routeNames') !== undefined && get(this, 'routeNames').length !== 0
+    return (get(this, 'routeNames') !== undefined && get(this, 'routeNames').length !== 0) ||
```

Comment:
The current convention for `get` against a component property is `this.get` - we're reserving `get(object, 'prop')` for non-`this` cases